emily_cron/app/services/deposit_processor.py

```diff
@@ -151,6 +226,10 @@ def update_deposits(self) -> None:
         locktime_updates = self.process_expired_locktime(enriched_deposits, bitcoin_chaintip_height)
         updates.extend(locktime_updates)
 
+        # Process RBF transactions
+        rbf_updates = self.process_rbf_transactions(enriched_deposits)
+        updates.extend(rbf_updates)
```

Comment:
Should we wait some blocks before marking the RBFd as failed? In case of reorgs, we would may endup with failed deposits that shouldn't be.